src/components/Tree/Tree.js

```diff
@@ -0,0 +1,162 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
What’s this preventing?

Reply:
The linter doesn't like the dependency arrays on useEffect how we are using them. I think I might end up disabling this rule project wide